BEP7.md

```diff
@@ -0,0 +1,320 @@
+  - [1. Summary](#1-summary)
+  - [2. Abstract](#2-abstract)
+  - [3. Status](#3-status)
+    - [Couple things to keep in mind](couple-things-to-keep-in-mind)
+  - [4. Motivation](#4-motivation)
+  - [5. Collection Specification](#5-collection-specification)
+    - [5.1 Collection Properties](#51-collection-properties)
+    - [5.2 Collection Management Operation](#52-collection-management-operation)
+      - [5.2.1 Issue collection](#521-issue-collection)
+      - [5.2.2 Increase total supply](#522-increase-total-supply)
+  - [6. Token Specification](#6-token-specification)
+    - [6.1 Token Properties](#61-token-properties)
+    - [6.2 Token Management Operation](#62-token-management-operation)
+      - [6.2.1 Issue token](#621-issue-token)
+      - [6.2.2 Transfer Tokens](#622-transfer-tokens)
+      - [6.2.3 Freeze Tokens](#623-freeze-tokens)
+      - [6.2.4 Unfreeze Tokens](#624-unfreeze-tokens)
+  - [7. Token Use Case Examples](#7-token-use-case-examples)
+  - [8. License](#8-license)
+  - [9. Reference](#9-reference)
+
+## 1. Summary
+
+This BEP describes a proposal for non-fungible tokens(NFTs) management on the Binance Chain.
+
+## 2. Abstract
+
+BEP-7 Proposal describes a common set of rules for non-fungible token within the Binance Chain ecosystem. It introduces the following details of a non-fungible token on Binance Chain:
+
+- What information makes a non-fungible token on Binance Chain
+- What actions can be performed on a non-fungible token on Binance Chain
+
+### Couple things to keep in mind
+
+- This BEP won't contain "how to implement" in terms of implmentation on the Binance Chain codebase, e.g. which module NFT should be in (this kind of discussion is happening on [cosmos-sdk/issues/4046](https://github.com/cosmos/cosmos-sdk/issues/4046)), because that should be happening in the PR(against to Binance Chain, when it's fully open source, so that dev can send out PR.
+
+- Apparently Binance chain doens't have solidity-such smart contract implemented, so don't expect to see many smart contract implmenations like what you can see in [EIP-721](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-721.md) in this BEP.
+
+## 3. Status
+
+This BEP is under drafting.
+
+## 4. Motivation
+
+Design and issue non-fungible asset on the Binance Chain, as one of the basic economic foundations of the blockchain.
+
+## 5. Collection Specification
+
+### 5.1 Collection Properties
+
+- Owner: the owner adress of the collection.
+
+- Token Name: Token Name represents the long name of the token - e.g. "BinanceAnnualEvent2019"
+
+- Denom: Denom is the identifier(symbol) of the newly issued collection.
+
+- Total Supply: Total supply is the total number of issued NFT tokens belong to this collection.
+
+- Mintable: A boolean value to indicate whether the owner can increase the totalSupply.
+
+- NFTs: An array contains items in NFT interface(described below).
+
```

Comment:
Should we include Token Index for an array of in game token assets for a multi-verse game like ready player one?
It's included in the ERC-1155 Spec

Reply:
You mean chainId? what is the Token index is referring to here?

Reply:
Yeah, @amazingandyyy chainID, this was from the last call last week @okwme mentioned the ERC-1155 Spec in the call.

Reply:
cool, yea not sure how binance plan to achieve cross-chain in the IBC/cosmos ecosystem in the future